src/generator/ruleDetailsSkeleton

```diff
@@ -0,0 +1,19 @@
+# _RULENAME_
+
+Checks if...
+
+## ❓ Why it's good to follow this rule?
+
+Content here...
+
+## 😱 Examples of code for which this rule will throw a warning
+
+Example of bad code...
+
+Extra details of the example... (if applicable)
+
+## 🤩 How to fix it?
+
+Example of how to fix it...
```

Comment:
I would add the ::: tip tag here

and an empty script tag with a todo comment

Reply:
Added in last commit